docs/datamodel.rst

```diff
@@ -167,6 +167,24 @@ with information about the case - signalling that *this entity* belongs to *this
 the intention of the FMU results data model to maintain *all* information about a case, and 
 in the future it is expected that ERT will serve case information beyond the basics.
 
+    **Dot-annotation** - we like it and use it. This is what it means:
+
+    The metadata structure is a dictionary-like structure, e.g.
+
+    .. code-block:: json
+
+        {
+            "myfirstkey": {
+                "mykey": "myvalue",
+                "anotherkey": "anothervalue"
+                }
+        }
+    
+    ::
```

Comment:
?? why ::

Reply:
Copy/paste, probably. I removed it and verified that (local) build still functions as intended.

Reply:
These intended blocks are meant as "info boxes" or whatever they are called. We could consider doing something on the styling side so that these get a different background or something to make them stand out. Currently, they are only intended a bit, but other than that look exactly the same as the rest of the text.